packages/database/src/TupaiaDatabase.js

```diff
@@ -407,45 +407,6 @@ export class TupaiaDatabase {
     return this.delete('setting', { key });
   }
 
-  /**
```

Comment:
This wasn't being used anywhere, and from the example appears to have been implemented for the user_reward use case specifically, so I thought it was best to delete it and keep the TupaiaDatabase as free of dead debris as possible